ph5/utilities/tests/test_segd2ph5.py

```diff
@@ -4,16 +4,17 @@
 import unittest
 import os
 import sys
+import shutil
+import tempfile
```

Comment:
Why add these? Are they used?

Reply:
I resolved this on the webpage. It was hard to check on the web. Then I pull to my computer to check. Check out the new commit